### PR TITLE
First branch

### PR DESCRIPTION
**Pull request by PatrykAdamus**
Please review my code

### CODE REVIEW

File: src/pl/adamus/patryk/git/learn/branch/BranchMain.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package pl.adamus.patryk.git.learn.branch;
+
+public class BranchMain {
+    public static void main(String[] args) {
+        System.out.println("Hello Branch");
+
```
Comment (juniorjavadeveloper-pl) [inline src/pl/adamus/patryk/git/learn/branch/BranchMain.java:6]: Please remove!

File: src/pl/adamus/patryk/git/learn/branch/BranchNextMain.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package pl.adamus.patryk.git.learn.branch;
+
+public class BranchNextMain {
+    public static void main(String[] args) {
+        System.out.println("Hello World");
```
Comment (juniorjavadeveloper-pl) [inline src/pl/adamus/patryk/git/learn/branch/BranchNextMain.java:5]: Add some code!

**Review by juniorjavadeveloper-pl — CHANGES_REQUESTED** [review]
Make changes!

File: src/pl/adamus/patryk/git/learn/branch/BranchMain.java (hunk 1 above)
Comment (PatrykAdamus) [reply src/pl/adamus/patryk/git/learn/branch/BranchMain.java:6]: Ok!

File: src/pl/adamus/patryk/git/learn/branch/BranchNextMain.java (hunk 2 above)
Comment (PatrykAdamus) [reply src/pl/adamus/patryk/git/learn/branch/BranchNextMain.java:5]: Ok

**Review by juniorjavadeveloper-pl — APPROVED** [review]
Looks good!

**Pull request merged**